src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/config/InterceptorConfig.java

```diff
@@ -41,10 +42,16 @@ public void addInterceptors(@NonNull InterceptorRegistry registry) {
                 .addPathPatterns(TRANSACTIONS);
         registry.addInterceptor(customUserAuthenticationInterceptor)
                 .addPathPatterns(TRANSACTIONS);
+        registry.addInterceptor(internalUserInterceptor());
```

Comment:
I think we need to do a bit more than this to get our interceptor code correct, now that transactions are being closed and filings are being sent. Might be best to revert these changes and create a new ticket for this?

Reply:
Removed the changes